<ui version="4.0" >
 <class>managerForm</class>
 <widget class="QDialog" name="managerForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>222</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Wii Connection Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resource.qrc" >:/actions/icons/actions/controller.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QTableWidget" name="tableWidget" >
       <property name="verticalScrollBarPolicy" >
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="dragDropMode" >
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="verticalScrollMode" >
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="gridStyle" >
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="rowCount" >
        <number>0</number>
       </property>
       <column>
        <property name="text" >
         <string>Connection name</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Hostname/IP</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Using</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Coment</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="selectButton" >
       <property name="text" >
        <string>Select</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <layout class="QGridLayout" >
     <item row="0" column="0" >
      <widget class="QPushButton" name="aConnectionButton" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="dConnectionButton" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="eConnectionButton" >
       <property name="text" >
        <string>Edit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource.qrc" />
 </resources>
 <connections/>
</ui>
